web_dialog_size/static/src/js/web_dialog_size_owl.esm.js

```diff
@@ -0,0 +1,62 @@
+/** @odoo-module **/
```

Comment:
Hi. Why do you put ``esm`` in the name of the file ? 
also, I think that putting ``owl`` is unnecessary. Don' you think ?

Reply:
`esm` is for being friendly with OWL linting. Please check the reference at https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-15.0

Reply:
@legalsylvain Talking about esm files, @pedrobaeza explained here, thanks. And without that, pre-commit is also returning errors, so it seems because of lintering. 

For another your question, did you mean why file name is having owl tag in name or why I used owl in here?

Reply:
Thanks for the esm explanation.

Regarding owl, my question was : why put owl in the name of the file.

Reply:
I put it because there is two files: one for older code, and for owl, it's like to separate where is old code, and where is the new. It's not necessary, I could rename without owl.